services/wallet/collectibles/manager.go

```diff
@@ -907,7 +907,18 @@ func (o *Manager) checkConnectionStatus(chainID walletCommon.ChainID) {
 			return
 		}
 	}
-	o.statuses[chainID.String()].SetIsConnected(false)
+
+	status, ok := o.statuses[chainID.String()]
+	if !ok {
+		status = connection.NewStatus()
+		state := status.GetState()
+		state.LastSuccessAt = time.Now().Unix()
```

Comment:
Not sure what we want to represent here, but the state we set is a bit weird: we say we've just successfully connected, but also `SetIsConnected(false)`

I guess the logical thing would be to just call `SetIsConnected` to false, which will modify `LastCheckedAt` accordingly.



- Leave connection status by default (`StateValueUnkown`)
- Leave `LastSuccessAt` by default (`InvalidTimestamp`)